### PR TITLE
FAQ about how to completely delete a host

### PR DESCRIPTION
**Pull request by weihanglo**
Related: https://github.com/harvester/harvester/issues/1497

### CODE REVIEW

**Comment by guangbochen** [comment]
also need to add to the https://docs.harvesterhci.io/v1.0/host/host/

**Comment by weihanglo** [comment]
How about move the whole section to **Host** doc?

**Comment by guangbochen** [comment]
Sure, let's move to the `Host` page, thanks.

File: docs/host/host.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
...
+## Deleting a Node
+
+To completely delete a node, you need to
+
+1. Go to the **Hosts** page.
+2. On the node you want to modify, click **⋮ > Delete**.
+3. Login to that node as root. Then run `rke2-uninstall.sh` to delete the whole RKE2 service.
+
+!!! warning
+    You will lost all data of the control plane on the node after deleing RKE2 service.
```
Comment (guangbochen) [inline docs/host/host.md:32]: - \s\lost\lose
- \s\RKE2\the RKE2

**Comment by guangbochen** [comment]
@weihanglo I also update the base branch to v1.0 but should also backport to the v0.3, thanks.

File: docs/host/host.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
...
+## Deleting a Node
+
+To completely delete a node, you need to
+
+1. Go to the **Hosts** page.
+2. On the node you want to modify, click **⋮ > Delete**.
+3. Login to that node as root. Then run `rke2-uninstall.sh` to delete the whole RKE2 service.
+
+!!! warning
+    You will lose all data of the control plane on the node after deleing the RKE2 service.
```
Comment (nunix) [inline docs/host/host.md:32]:
```suggestion
    You will lose all data of the control plane node after deleing the RKE2 service.
```

File: docs/host/host.md [hunk 3]
```diff
@@ -20,6 +20,23 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
 
 ![cordon-node.png](./assets/cordon-nodes.png)
 
+## Deleting a Node
+
+To completely delete a node, you need to
+
+1. Go to the **Hosts** page.
+2. On the node you want to modify, click **⋮ > Delete**.
+3. Login to that node as root. Then run `rke2-uninstall.sh` to delete the whole RKE2 service.
```
Comment (nunix) [inline docs/host/host.md:29]:
```suggestion
Deleting a node is done in two phases:

1. Delete the node from Harvester
    - Go to the **Hosts** page
    - On the node you want to modify, click **⋮ > Delete**

2. Uninstall RKE2 from the node
    - Login to the node as root
    - Run `rke2-uninstall.sh` to delete the whole RKE2 service.
```

File: docs/host/host.md [hunk 4]
```diff
@@ -20,6 +20,23 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
...
+
+!!! warning
+    You will lose all data of the control plane on the node after deleing the RKE2 service.
+
+!!! note
+    This is a [known issue](https://github.com/harvester/harvester/issues/1497) about node hard delete.
+    Once it is resolved. The last step can be skipped.
```
Comment (nunix) [inline docs/host/host.md:36]:
```suggestion
    There's a [known issue](https://github.com/harvester/harvester/issues/1497) about node hard delete.
    Once resolved, the last step can be skipped.
```

File: docs/terraform/terraform.md [hunk 5]
```diff
@@ -7,14 +7,28 @@
...
-
-- Use the following command to complete the above actions:
-```
-tar -zxvf terraform-provider-harvester-amd64.tar.gz
-./install-terraform-provider-harvester.sh
+### **Option 1:** Download and install the provider from the Terraform [registry](https://registry.terraform.io/providers/harvester/harvester/latest). 
+
+To install this provider, copy and paste this code into your Terraform configuration. Then, run terraform init.
+
+Terraform 0.13+
```
Comment (nunix) [inline docs/terraform/terraform.md:14]:
```suggestion
```

Comment (nunix) [reply docs/terraform/terraform.md:14]: This line can be removed as the minimal Terraform version is mentioned in the `Requirements` section

**Review by nunix — COMMENTED** [review]
Rephrased the description, `warning` and `note` for the node deletion

Removed duplicated content in `terraform` page

File: docs/terraform/terraform.md (hunk 5 above)
Comment (weihanglo) [reply docs/terraform/terraform.md]: Sorry this change is not in this PR. I've updated the target branch. Thanks!

**Comment by weihanglo** [comment]
@nunix Updated accordingly. Thanks!

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**